src/main/java/org/sefglobal/scholarx/service/EmailService.java

```diff
@@ -1,37 +1,32 @@
 package org.sefglobal.scholarx.service;
-import org.sefglobal.scholarx.model.Email;
-import org.sefglobal.scholarx.util.EmailUtil;
+
+import org.sefglobal.scholarx.model.Mail;
 import org.springframework.stereotype.Service;
 
-import javax.mail.MessagingException;
 import java.io.IOException;
 import java.util.HashMap;
 import java.util.Map;
 
 @Service
 public class EmailService {
 
-    private final EmailUtil emailUtil;
-
-    public EmailService(EmailUtil emailUtil) {
-        this.emailUtil = emailUtil;
-    }
+    private SimpleJavaMail simpleJavaMail;
 
-    public Email sendEmail(String emailAddress, String subject, String message, boolean showButton) throws IOException, MessagingException {
-        Email email = new Email();
+    public void sendEmail(String name, String emailAddress, String subject, String message, boolean showButton) throws IOException {
+        Mail email = new Mail();
+        email.setName(name);
         email.setEmail(emailAddress);
         email.setSubject(subject);
         email.setMessage(message);
 
         Map<String, Object> model = new HashMap<String, Object>();
+        model.put("name", name);
         model.put("emailAddress", emailAddress);
         model.put("subject", subject);
         model.put("message", message);
         model.put("showButton", showButton);
         email.setProps(model);
 
-        emailUtil.sendSimpleMessage(email);
-
-        return email;
+        simpleJavaMail.sendEmail(email);
```

Comment:
We should use the singleton class here, right?

Reply:
Here, the email object will pass for the simple java mail class, and there it will get a connection from the mailer instance (from the singleton class). @jayasanka-sack

Reply:
No need. Send the email here itself.